### ISSUE TITLE
Redesign Home Page

### ISSUE THREAD

**Issue by ajaynegi45** [ajaynegi45/LibraryMan#81]
### Create a New Home Page Design
- Attach the new design below
- When the book icon in the bottom navbar is clicked, redirect users to the "/books" page
- Ensure the design is:
     - Responsive for screen sizes between 320px and 1440px
     - Optimized for small mobile devices

![Home Page](https://github.com/user-attachments/assets/7c0cf33e-b020-49a3-87bb-3997f0a45b9d)

**Comment by Bhargav02550** [comment]
Is this a mobile application ui for a web ui

**Comment by Bhargav02550** [comment]
I want to work on this issue

**Comment by ajaynegi45** [comment]
> Is this a mobile application ui for a web ui

@Bhargav02550 This is web UI

**Comment by ajaynegi45** [comment]
> I want to work on this issue

Hi @Bhargav02550,

Thank you for expressing your interest in working on the "Redesign Home Page" issue. I'm delighted to inform you that I have assigned this issue to you. Your willingness to contribute to our project and "Redesign Home Page" on Website is much appreciated.

Feel free to start working, and if you have any questions or need assistance during the process, please don't hesitate to reach out. We eagerly anticipate your work on creating an informative and visually appealing "Home Page" for our website.

**Comment by Bhargav02550** [comment]
So in mobile ratio u want a bottom bar right

**Comment by Bhargav02550** [comment]
I'm new to contributions can you follow me the procces of contributing to the project if you don't mind

**Comment by ajaynegi45** [comment]
> I'm new to contributions can you follow me the procces of contributing to the project if you don't mind

Here's a refined version:

---

@Bhargav02550 Absolutely! If you encounter any issues or have questions while contributing, don't hesitate to reach out—I’m here to help!

Please take a moment to go through the following:
- [Code of Conduct](https://github.com/ajaynegi45/LibraryMan/blob/main/code_of_conduct.md)
- [README](https://github.com/ajaynegi45/LibraryMan/blob/main/README.md)
- [Contributing Guidelines](https://github.com/ajaynegi45/LibraryMan/blob/main/contributing.md)

**Comment by ajaynegi45** [comment]
> I'm new to contributions can you follow me the procces of contributing to the project if you don't mind

@Bhargav02550 Absolutely! If you encounter any issues or have questions while contributing, don't hesitate to reach out—I’m here to help!

Please take a moment to go through the following:
- [Code of Conduct](https://github.com/ajaynegi45/LibraryMan/blob/main/code_of_conduct.md)
- [README](https://github.com/ajaynegi45/LibraryMan/blob/main/README.md)
- [Contributing Guidelines](https://github.com/ajaynegi45/LibraryMan/blob/main/contributing.md)

**Comment by Bhargav02550** [comment]
Hey negi do you want that ui in web when it is in mobile ratio right

**Comment by ajaynegi45** [comment]
> Hey negi do you want that ui in web when it is in mobile ratio right

@Bhargav02550 For your reference, my name is Ajay Negi, not just Negi. I want the UI to be optimized for both tablet and mobile devices. For larger screens, please shift the navbar to the top, remove the icon, and write the name instead. Fixed the  size min 320px - max 1440px

**Comment by ajaynegi45** [comment]
> Hey negi do you want that ui in web when it is in mobile ratio right

@Bhargav02550 
### Checkout Design Resources:
- <a href="https://www.figma.com/design/kUaU4vgMuqVi2lDgWlsKhk/LibraryMan-UI?node-id=11-2&t=aD7uCyRzmhbGQsBv-1" target="_blank">Figma Design</a>

**Comment by Bhargav02550** [comment]
![Screenshot 2024-10-02 222833](https://github.com/user-attachments/assets/93fd312c-b1d0-4a1e-8393-d2d3ee2e63fb)
Hey Ajay How is this app bar looking please share your opinion. Looking forward to work on based on your response

**Comment by ajaynegi45** [comment]
> ![Screenshot 2024-10-02 222833](https://private-user-images.githubusercontent.com/99163827/372959677-93fd312c-b1d0-4a1e-8393-d2d3ee2e63fb.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3Mjc4ODk2MjcsIm5iZiI6MTcyNzg4OTMyNywicGF0aCI6Ii85OTE2MzgyNy8zNzI5NTk2NzctOTNmZDMxMmMtYjFkMC00YTFlLTgzOTMtZDJkM2VlMmU2M2ZiLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDEwMDIlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMDAyVDE3MTUyN1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWI0MGMwMjU2MjllMzdlYTEzOGE2MzU3OWE5ZGNjYzJiNzE2ZTU0Y2I5ZmUyNGRkMWU4YjJlOTNiNzcwNzdlYTAmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.bdvxmen5SPv8LrASTzjxQT1fsnUS6cQTTcE0_9v0g_8) Hey Ajay How is this app bar looking please share your opinion. Looking forward to work on based on your response

The app bar looks great! Keep up the good work. If you have any questions or need guidance on the next steps, don't hesitate to reach out. We're excited to see your contributions!

**Comment by Bhargav02550** [comment]
![Screenshot 2024-10-03 223907](https://github.com/user-attachments/assets/049d9eee-f9fe-4951-a70f-c5ff577f8f08)
Hey @ajaynegi45 look at this tell me if any enhancements needed i will add the colors that you gave in the figma file Please share you opinion and is there any ui design for screens with ration above 768 pixels

**Comment by ajaynegi45** [comment]
> ![Screenshot 2024-10-03 223907](https://private-user-images.githubusercontent.com/99163827/373348104-049d9eee-f9fe-4951-a70f-c5ff577f8f08.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3Mjc5NzY0NTEsIm5iZiI6MTcyNzk3NjE1MSwicGF0aCI6Ii85OTE2MzgyNy8zNzMzNDgxMDQtMDQ5ZDllZWUtZjlmZS00OTUxLWE3MGYtYzVmZjU3N2Y4ZjA4LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDEwMDMlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMDAzVDE3MjIzMVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTU1ZjQ5NThlYzg0OWJlZDVhZWIyNDI2MDQ5ZDc3ZWI0Mjc2YjFjOWI3OGI1MDZhMTg0NWM0MTkzZjVhMzkxZGImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.IjixkqKaCO3X_ly1jZG63w6E2JmQhcb2B9gmvqfRrIM) Hey @ajaynegi45 look at this tell me if any enhancements needed i will add the colors that you gave in the figma file Please share you opinion and is there any ui design for screens with ration above 768 pixels

Remove the top navigation and align it with the Figma design. For screens below 768 pixels, make it resemble an app, and for screens above 768 pixels, make it look like a website. Remove the bottom navigation and set the body width to 1200px.

**Comment by Bhargav02550** [comment]
Hey @ajaynegi45 are you specific to colors that you mentioned in the figma file

**Comment by ajaynegi45** [comment]
> Hey @ajaynegi45 are you specific to colors that you mentioned in the figma file

Yes, But if you have any better colour. You can show me

**Comment by Bhargav02550** [comment]
![image](https://github.com/user-attachments/assets/31c2c6a6-1e79-4f94-bc0a-1fdc2bf5cb01)
Hey @ajaynegi45 struck with the navbar kind off confusion in the figma design help me out.
